connector/dojot_acl/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file index.js
+ * @dependencies [ util, @dojot/microservice-sdk ]
+ */
+
+const util = require('util');
+const { ConfigManager, Logger } = require('@dojot/microservice-sdk');
+
+const DOJOT_ACL_CONFIG_LABEL = 'DOJOT_ACL';
+
+const userConfigFile = process.env.DOJOT_ACL_APP_USSER_CONFIG_FILE || 'production.conf';
+
+ConfigManager.loadSettings(DOJOT_ACL_CONFIG_LABEL, userConfigFile);
+
+const config = ConfigManager.getConfig(DOJOT_ACL_CONFIG_LABEL);
+
+Logger.setTransport('console', { level: config.log['console.level'] });
```

Comment:
To follow the other services in this repository, you may be able to configure options for log in  files.